modules/virt-about-cluster-checkup-framework.adoc

```diff
@@ -10,4 +10,9 @@ A _checkup_ is an automated test workload that allows you to verify if a specifi
 
 By using predefined checkups, cluster administrators can improve cluster maintainability, troubleshoot unexpected behavior, minimize errors, and save time. They can also review the results of the checkup and share them with experts for further analysis. Vendors can write and publish checkups for features or services that they provide and verify that their customer environments are configured correctly.
 
-Running a predefined checkup in the cluster involves setting up the namespace and service account for the framework, creating the `ClusterRole` and `ClusterRoleBinding` objects for the service account, enabling permissions for the checkup, and creating the input config map and the checkup job. You can run a checkup multiple times. 
+Running a predefined checkup in the cluster involves setting up the namespace and service account for the framework, creating the `ClusterRole` and `ClusterRoleBinding` objects for the service account, enabling permissions for the checkup, and creating the input config map and the checkup job. You can run a checkup multiple times.
+
+[WARNING]
+====
+Before enabling permissions that the checkup requires for cluster access, you must verify that the checkup image is from a trustworthy source. This is because the framework automatically binds the `ClusterRole` objects provided by the checkup vendor to grant permissions to the checkup instance.
```

Comment:
What does "enabling permissions" mean?

Reply:
@orelmisan I have reworded the note per your suggestion.